### DISCUSSION TITLE
Fails to run commands

### DISCUSSION THREAD

**Discussion by ksamirdev** [vercel/turborepo#6981] [Help]
I'm getting `ERROR  run failed: root task //#build (turbo run build) looks like it invokes turbo and might cause a loop` error in turbo with nextjs 14. I installed turbo with `yarn add turbo --dev --ignore-workspace-root-check` 

and my `turbo.json` looks like:
```json
{
  "$schema": "https://turbo.build/schema.json",
  "globalDependencies": ["**/.env.*local"],
  "pipeline": {
    "build": {
      "dependsOn": ["^build"],
      "outputs": [".next/**", "!.next/cache/**"]
    },
    "lint": {
      "dependsOn": ["^lint"]
    },
    "dev": {
      "cache": false,
      "persistent": true
    }
  }
}
```

Turbo version: `^1.11.3`
Yarn version: `1.22.21`
OS: `Ubuntu 23.10 x86_64`
CPU: `DO-Premium-Intel (2) @ 2.494GHz`
GPU: `00:02.0 Red Hat, Inc. Virtio 1.0 GPU`

**Comment by mehulkar** [comment]
What are you running in your shell to get that error?

**Reply by ksamirdev** [reply]
Just the build command. `turbo run build`

**Reply by mehulkar** [reply]
What does your `package.json`'s `build` command look like?

**Comment by mehulkar** [comment]
Closing as there has been no reply for a while, feel free to ping if we need to follow up more!
